src/app/components/video/index.js

```diff
@@ -1,18 +1,33 @@
 import React from 'react';
-import MediaQuery from 'react-responsive';
-
+import classnames from 'classnames';
 import Rimage from 'app/components/rimage';
 
 const posterURL = "/images/transparent.png";
+
 const Video = React.createClass({
+
+  getInitialState() {
+    return {
+      mobile: false
```

Comment:
🤔 I don't think this is state. It should be a props the component uses to render accordingly. So, the value `window.innerWidth` or the flag `mobile` should be computed outside (in the home component I guess). That means, less scattered state.